syntax = "proto3";

package vonk_fence;

import "shared.proto";

message CameraOut {
    optional PhotoResult photo_result = 1;
    optional RegionOfInterest region_of_interest = 2;
}

message PhotoResult {
    string filename = 1;
    uint32 time_taken_millis = 2;
    optional string well_known = 3;
}
